### DISCUSSION TITLE
How to access child widgets

### DISCUSSION THREAD

**Discussion by busslina** [GregoryConrad/rearch-dart#77] [Q&A] [answered]
With `StatefulWidget` we can use a `GlobalKey` to access child states down the widget tree. Is there an analogous way to do this with `RearchConsumer`?

I try to use this mechanism the less I can, but sometimes I'm forced to. E.G. requesting focus to one of the children (Tabs).

**Comment by GregoryConrad** [comment, marked as answer]
> to access child states down the widget tree

Like in #75, you shouldn't be using `GlobalKey` for this; use scoped state with an `InheritedWidget` instead (at least until I can make the macro to cut out the RearchConsumer + InheritedWidget boilerplate).

However, there currently isn't a side effect for focus nodes. Tracking that via #78
